README.md

```diff
@@ -262,34 +259,34 @@ Escolher uma das opções a seguir, definidas no arquivo *internal/errors/errors
 - EINVALID // validation failed - Erros de lógica criadas por nós. Exemplo: salvar um usuário no banco de dados
 - ENOTFOUND // entity does not exist
 - EFORBIDDEN //operation forbidden
-- EEXPECTED //expected error that don't need to be logged. 
+- EEXPECTED //expected error that don't need to be logged.
 
 **Exemplos de erros:**
 
 ```go
 //Find address na camada de repositório
 func (r *MongoRepository) Find(id entity.ID) (*entity.Address, error) {
-	result := entity.Address{}
-	session := r.pool.Session(nil)
-	defer session.Close()
-	coll := session.DB(r.db).C("address")
-	err := coll.Find(bson.M{"_id": id}).One(&result)
-
-	if err != nil {
-		return nil, &errors.Error{Op: "address.MongoRepository.Find", Err: err, Code: errors.ENOTFOUND}
-	}
-	return &result, nil
+ result := entity.Address{}
```

Comment:
@avelino are these changes expected? I believe that you wanna remove empty spaces but I believe it should not happen here.

Reply:
my editor switched automatically, sorry